### DISCUSSION TITLE
currency in inventory

### DISCUSSION THREAD

**Discussion by Denzzzb** [TownyAdvanced/Towny#5744] [Ideas]
Apologies in advance for the correctness of my English (Google translate)
Towny has long been a legendary project that continues to delight users! The plugin makes a huge contribution to the RP of any server. But the digital currency, in my opinion, spoils the rp process a little and has to be turned off, breaking half of the plugin's functions
I understand that this is difficult to implement, but if it works, it will be a real breakthrough!
Of course, there are economic plugins that are not related to Towny, which can do this to some extent, but this does not work correctly and can crash the server.
I beg you to treat this proposal with understanding and think over it. I can describe in more detail any things or ideas if you want.

**Comment by LlmDl** [comment]
It's nice to hear you appreciate Towny, perhaps you'll become a sponsor one day.

Towny will not add a physical currency feature, because Towny is not an economy plugin. You can use TheNewEconomy or Gringott's or other Vault-ready Economy plugins. If you're experiencing server crashes from your economy plugin, you should file a ticket with the author of your economy plugin.

**Reply by Denzzzb** [reply]
Your plugin has a currency, isn't it possible to make its equivalent, but in material form? I do not need a global currency for the entire server, but only yours in order to buy plots and take taxes, but with the help of material currency, not electronic

**Reply by LlmDl** [reply]
My plugin does not have a currency, it uses your economy plugin's currency.

**Reply by creatorfromhell** [reply]
> Your plugin has a currency, isn't it possible to make its equivalent, but in material form? I do not need a global currency for the entire server, but only yours in order to buy plots and take taxes, but with the help of material currency, not electronic

Can support statements, you're probably utilizing a default economy plugin such as essentials.

**Reply by Denzzzb** [reply]
![image](https://user-images.githubusercontent.com/99116188/154216779-df9d0c09-8f83-4811-96b3-994730e283eb.png)
in any case, I was interested in the TNE plugin, I will test with it

**Reply by LlmDl** [reply]
The Economy Settings section is how you configure how Towny interacts with your economy plugin. The New Economy is probably your best route to achieving a physical economy.
